neqo-transport/src/connection/tests/priority.rs

```diff
@@ -386,7 +386,7 @@ fn low() {
     // Send a session ticket and make it big enough to require a whole packet.
     // The resulting CRYPTO frame beats out the stream data.
     let stats_before = server.stats().frame_tx;
-    server.send_ticket(now, &[0; 2048]).unwrap();
+    let _ = server.send_ticket(now, &vec![0; server.plpmtu().unwrap()]);
```

Comment:
Why can a potential `Err` be ignored here?

Reply:
It's a test, so I assumed a panic is OK if there is an issue.